Slash/DB/MySQL/MySQL.pm

```diff
@@ -13646,7 +13652,22 @@ PRIMARY KEY (discussion_id, uid)
 		if (!$self->sqlDo("ALTER TABLE users_comments DROP mode")) {
                         return 0;
                 }
-		if (!$self->sqlDo("ALTER TABLE users_comments CHANGE mode_new mode ENUM('flat', 'nocomment', 'thread') NOT NULL DEFAULT 'thread'")) {
+		if (!$self->sqlDo("ALTER TABLE users_comments CHANGE mode_new mode ENUM('flat', 'nocomment', 'thread', 'threadtos', 'threadtng') NOT NULL DEFAULT 'thread'")) {
```

Comment:
Instead of this, at this point we should do this: 

if (!$self->sqlDo("update users_comments set mode = 'threadtos' where mode_new = 'thread'")) {
                        return 0;
               }

if (!$self->sqlDo("ALTER TABLE users_comments ADD mode ENUM('flat', 'nocomment', 'threadtng', 'threadtos') NOT NULL DEFAULT 'threadtos'")) {
                        return 0;
                }

if (!$self->sqlDo("UPDATE users_comments SET mode = mode_new")) {
                        return 0;
                }

                if (!$self->sqlDo("ALTER TABLE users_comments DROP mode_new")) {
                        return 0;
                }